### ISSUE TITLE
Publishing TADA via CRAN and ROpenSci

### ISSUE THREAD

**Issue by cristinamullin** [USEPA/TADA#452]
**Is your feature request related to a problem? Please describe.**

Let's work on getting TADA published via CRAN. Then we can also publish it via ROpenSci.

Note from Justin b: pyOpenSci is actually modeled after ROpenSci, I think Jeff Holiister and Laura DeCicco are both editors for it. CRAN will make it more visible/accessible, ROpenSci will help with code standards and infrastructure that make it more maintainable. You’re already using a lot of that infrastructure but a worthwhile review after CRAN. Let me know if I can be helpful on either front.

[https://ropensci.org/software-review/](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fropensci.org%2Fsoftware-review%2F&data=05%7C02%7CBousquin.Justin%40epa.gov%7C01b0b08e240a423a191b08dc5b171f54%7C88b378b367484867acf976aacbeca6a7%7C0%7C0%7C638485402061134370%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=VZOmjNPWKTPp06li%2F%2BsgBY5Flnd54ajRZ66oTHUHxwE%3D&reserved=0)

**Describe the solution you'd like**

Let's use this issue to plan next steps and track progress on a CRAN submission.

**Comment by cristinamullin** [comment]
To start, we must address all issues, including notes produced by this more robust CRAN check:
```r
devtools::check(manual = TRUE, remote = TRUE, incoming = TRUE)

**Comment by cristinamullin** [comment]
We must pick a new name for the TADA package, because tada is already a taken/published package on CRAN: 
![image](https://github.com/USEPA/TADA/assets/46969696/3b9e1cce-7aa6-4a28-a967-aed176987e90)

**Comment by erg-hzhou** [comment]
I suggest you consider using EPATADA as the package name in CRAN. See an example - https://cran.r-project.org/web/packages/epanet2toolkit/index.html

**Comment by cristinamullin** [comment]
I agree and have implemented this change in the cm-release-review-2 branch for both the package and shiny app. Plan to merge in next week after team review.
